haystack/utils/doc_store.py

```diff
@@ -10,22 +10,40 @@
 WEAVIATE_CONTAINER_NAME = "weaviate"
 
 
-def launch_es(sleep=15, delete_existing=False, java_opts: Optional[str] = None):
+def launch_es(
```

Comment:
I would make this a generic private `_launch_es()` and have two separate `launch_es7()` and `launch_es8()` call that. 

It think it's clearer and less prone to error, it's way to easy to forget setting the `version_tag`.

Reply:
What do you think about instead logging warning message if the installed es client version doesn't match the version tag?